pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -173,21 +170,25 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
           </div>
 
           <div className={styles.cardsContainer}>
-            <OrganizationCard
-              title={{ label: organization.name, href: `/organizations/${organization.id}` }}
-              description={organization.description}
-              website={organization.website}
-              logo={organization.logo}
-              components={{
-                owned: organization.owns?.length.toString() ?? "0",
-                supported: organization.supports?.length.toString() ?? "0",
-                used: organization.uses?.length.toString() ?? "0",
-              }}
-              gitHub={organization.github}
-              gitLab={organization.gitlab}
-              type={organization.type}
-              layoutClassName={styles.organizationCardContainer}
-            />
+            {organizationData ? (
+              <OrganizationCard
+                title={{ label: organizationData.name, href: `/organizations/${organizationData.id}` }}
+                description={organizationData.description}
+                website={organizationData.website}
+                logo={organizationData.logo}
+                components={{
+                  owned: organizationData.owns?.length.toString() ?? "0",
+                  supported: organizationData.supports?.length.toString() ?? "0",
+                  used: organizationData.uses?.length.toString() ?? "0",
+                }}
+                gitHub={organizationData.github}
+                gitLab={organizationData.gitlab}
+                type={organizationData.type}
+                layoutClassName={styles.organizationCardContainer}
+              />
+            ) : (
+              <div className={styles.organizationCardContainer} />
```

Comment:
I think it would be better if we put a `<span>` in this empty space with "There is no organization available" in it to fill up the space and tell the user why there is so much space there